### DISCUSSION TITLE
Does GraphScope only support int type as vertex `id`?

### DISCUSSION THREAD

**Discussion by parkerzf** [alibaba/GraphScope#1747] [Q&A] [answered]
I try to load the graph with string type vertex id but get an exception. Is it expected?

**Comment by sighingnow** [comment, marked as answer]
You need speicify the `oid_type` as `string`:

https://github.com/alibaba/GraphScope/blob/7af26c96c64502be6abcdd762752e7c6bac8680b/python/graphscope/tests/unittest/test_create_graph.py#L228-L237

**Comment by parkerzf** [comment]
Thanks! will try
